### PR TITLE
Yatri logo change

### PR DESCRIPTION
**Pull request by MercyQueen**
## Type of Change

- [x] Bugfix
- [ ] New feature
- [ ] Enhancement
- [ ] Refactoring
- [ ] Dependency updates

### CODE REVIEW

File: Frontend/ui-customer/src/Components/StepsHeaderModel/View.purs [hunk 1]
```diff
@@ -13,7 +13,7 @@ import Data.Maybe as Maybe
 import Common.Types.App (LazyCheck(..))
 
 view :: forall w . (Action -> Effect Unit) -> StepsHeaderModelState -> PrestoDOM (Effect Unit) w
-view push state = 
+view push state = do
```
Comment (Vignesh-772) [inline Frontend/ui-customer/src/Components/StepsHeaderModel/View.purs]: no need of `do` here

File: Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/Controller.purs [hunk 2]
```diff
@@ -2003,16 +2010,40 @@ estimatesFlow estimatedQuotes state = do
...
       $ SelectEstimate
           state
-            { data { suggestedAmount = estimatedPrice, rateCard = { baseFare: baseFare, extraFare: extraFare, pickUpCharges: pickUpCharges, additionalFare: additionalFare, nightShiftMultiplier: nightShiftMultiplier, nightCharges: nightCharges,currentRateCardType: DefaultRateCard,onFirstPage: false}, showPreferences = getPreferenceValue "" }
+            { data  { suggestedAmount = estimatedPrice, 
+                      rateCard =  { additionalFare: additionalFare
+                                  , nightShiftMultiplier: nightShiftMultiplier
+                                  , nightCharges: nightCharges
+                                  , currentRateCardType: DefaultRateCard
+                                  , onFirstPage: false
+                                  , rateCardArray : ([ { title : (getString MIN_FARE_UPTO) <> if nightCharges then " 🌙" else "", description : "₹" <> toString (baseFare) }
```
Comment (Vignesh-772) [inline Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/Controller.purs:2019]: create a function to get this array

File: Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/Controller.purs [hunk 3]
```diff
@@ -2003,16 +2010,40 @@ estimatesFlow estimatedQuotes state = do
...
     continue
       state
         { props { currentStage = SearchLocationModel, rideRequestFlow = false, isSearchLocation = SearchLocation, isSrcServiceable = true, isDestServiceable = true, isRideServiceable = true, showRateCardIcon = showRateCardIcon }
-        , data { rateCard = { baseFare: baseFare, extraFare: extraFare, pickUpCharges: pickUpCharges, additionalFare: additionalFare, nightShiftMultiplier: nightShiftMultiplier, nightCharges: nightCharges,currentRateCardType: DefaultRateCard,onFirstPage: false} }
+        , data { rateCard = { additionalFare: additionalFare
+                            , nightShiftMultiplier: nightShiftMultiplier
+                            , nightCharges: nightCharges
+                            , currentRateCardType: DefaultRateCard
+                            , onFirstPage: false
+                            , rateCardArray : ([ { title :( if (lang == "EN_US") then (getString MIN_FARE_UPTO) <> " 2 km" else "2 km " <> (getString MIN_FARE_UPTO) ) <> if nightCharges then " 🌙" else "" , description : "₹" <> toString (baseFare) }
```
Comment (Vignesh-772) [inline Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/Controller.purs:2040]: same here

**Review by Vignesh-772 — CHANGES_REQUESTED** [review]

File: Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/Controller.purs [hunk 4]
```diff
@@ -2121,4 +2160,42 @@ getZoneType :: Maybe String -> ZoneType
...
+
+    "SUV"       -> [ { title : if lang == "EN_US" then (getString MIN_FARE_UPTO) <> " 5 km" else "5 km " <> (getString MIN_FARE_UPTO) , description : "₹165"}
+                   , { title : "5 km - 13 km" , description : "₹20 / km"}
+                   , { title : "13 km - 30 km" , description : "₹28 / km"}
+                   , { title : if lang == "EN_US" then (getString MORE_THAN) <> " 30 km" else "30 " <> (getString MORE_THAN) , description :"₹40 / km"}
+                   , { title : (getString PICKUP_CHARGE), description : "₹" <> (show state.data.pickUpCharges) }
+                   , { title : (getString DRIVER_ADDITIONS) ,description : "₹0 - ₹60"}]
+    _ -> []
+
+getRateCardArray :: Boolean -> String -> Int -> Int -> Int -> Array {title :: String , description :: String}
```
Comment (Vignesh-772) [inline Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/Controller.purs:2198]: Give proper type signature

**Review by Vignesh-772 — APPROVED** [review]

File: Frontend/ui-common/src/Components/ChooseVehicle/View.purs [hunk 5]
```diff
@@ -5,9 +5,11 @@ import Common.Types.App
 import Components.ChooseVehicle.Controller (Action(..), Config)
 import Effect (Effect)
 import Font.Style as FontStyle
-import Prelude (Unit, const, ($), (<>), (==), (&&), not)
+import Prelude (Unit, const, ($), (<>), (==), (&&), not, pure, unit)
 import PrestoDOM (Gravity(..), Length(..), Margin(..), Orientation(..), Padding(..), PrestoDOM, Visibility(..), background, clickable, color, cornerRadius, gravity, height, imageView, imageWithFallback, linearLayout, margin, onClick, orientation, padding, relativeLayout, stroke, text, textView, visibility, weight, width)
 import Common.Styles.Colors as Color
+import Debug (spy)
```
Comment (kavya-shree-s) [inline Frontend/ui-common/src/Components/ChooseVehicle/View.purs:11]: Remove spy import

File: Frontend/ui-customer/src/Screens/CustomerUtils/MyProfileScreen/Controller.purs [hunk 6]
```diff
@@ -111,12 +111,12 @@ eval (UserProfile (GetProfileRes profile)) state = do
   continue state { data { name = name, editedName = name, gender = gender, emailId = profile.email} }
 eval (NameEditTextAction (PrimaryEditText.TextChanged id value)) state = do
   _ <- pure $ spy "Value changed"  value
```
Comment (kavya-shree-s) [inline Frontend/ui-customer/src/Screens/CustomerUtils/MyProfileScreen/Controller.purs:113]: Remove spy statement

**Review by kavya-shree-s — CHANGES_REQUESTED** [review]

**Comment by kavya-shree-s** [comment]
@MercyQueen Resolve conflict and squash the commit

**Review by witcher-shailesh — APPROVED** [review]

File: Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/Controller.purs [hunk 7]
```diff
@@ -2121,4 +2160,42 @@ getZoneType :: Maybe String -> ZoneType
...
+                   , { title : if lang == "EN_US" then (getString MORE_THAN) <> " 30 km" else "30 " <> (getString MORE_THAN) ,description : "₹36 / km"}
+                   , { title : (getString PICKUP_CHARGE), description : "₹" <> (show state.data.pickUpCharges) }
+                   , { title : (getString DRIVER_ADDITIONS) ,description : "₹0 - ₹60"}]
+
+    "SUV"       -> [ { title : if lang == "EN_US" then (getString MIN_FARE_UPTO) <> " 5 km" else "5 km " <> (getString MIN_FARE_UPTO) , description : "₹165"}
+                   , { title : "5 km - 13 km" , description : "₹20 / km"}
+                   , { title : "13 km - 30 km" , description : "₹28 / km"}
+                   , { title : if lang == "EN_US" then (getString MORE_THAN) <> " 30 km" else "30 " <> (getString MORE_THAN) , description :"₹40 / km"}
+                   , { title : (getString PICKUP_CHARGE), description : "₹" <> (show state.data.pickUpCharges) }
+                   , { title : (getString DRIVER_ADDITIONS) ,description : "₹0 - ₹60"}]
```
Comment (ashishsingh101) [inline Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/Controller.purs:2195]: why are we hardcoding these values ?

File: Frontend/ui-driver/src/Flow.purs [hunk 8]
```diff
@@ -80,6 +80,7 @@ baseAppFlow baseFlow = do
     checkVersion versionCode
     cacheAppParameters versionCode
     when baseFlow $ void $ UI.splashScreen state.splashScreen
+    setValueToLocalNativeStore NEGOTIATION_UNIT if (getMerchant unit == YATRIPARTNER) then "20" else "10"
```
Comment (ashishsingh101) [inline Frontend/ui-driver/src/Flow.purs:83]: do we need `NEGOTIATION_UNIT ` ?

**Review by ashishsingh101 — CHANGES_REQUESTED** [review]

File: Frontend/ui-driver/src/Flow.purs (hunk 8 above)
Comment (MercyQueen) [reply Frontend/ui-driver/src/Flow.purs:83]: yes this is required for yatri partner. We are setting the value by which the driver can increment the price as 20 as we are taking it from shared prefs.

**Pull request merged**